core/src/com/unciv/ui/MultiplayerScreen.kt

```diff
@@ -0,0 +1,372 @@
+    private val addGameText = "Add Multiplayer Game".tr()
+    private val copyGameIdText = "Copy Game ID".tr()
+    private val copyUserIdText = "Copy User ID".tr()
+    private val refreshText = "Refresh List".tr()
+
+    private val editButton = TextButton(editButtonText, skin).apply { disable() }
+    private val addGameButton = TextButton(addGameText, skin)
+    private val copyGameIdButton = TextButton(copyGameIdText, skin).apply { disable() }
+    private val copyUserIdButton = TextButton(copyUserIdText, skin)
+    private val refreshButton = TextButton(refreshText, skin)
+
+    init {
+        setDefaultCloseAction()
+
+        //Help Button Setup
+        val tab = Table()
+        val helpButton = TextButton("?", skin)
+        helpButton.onClick {
+            val helpPopup = Popup(this)
+            helpPopup.addGoodSizedLabel("To create a multiplayer game, check the 'multiplayer' toggle in the New Game screen, and for each human player insert that player's user ID.".tr()).row()
+            helpPopup.addGoodSizedLabel("You can assign your own user ID there easily, and other players can copy their user IDs here and send them to you for you to include them in the game.".tr()).row()
+            helpPopup.addGoodSizedLabel("").row()
+
+            helpPopup.addGoodSizedLabel("Once you've created your game, the Game ID gets automatically copied to your clipboard so you can send it to the other players.".tr()).row()
+            helpPopup.addGoodSizedLabel("Players can enter your game by copying the game ID to the clipboard, and clicking on the 'Add Multiplayer Game' button".tr()).row()
+            helpPopup.addGoodSizedLabel("").row()
+
+            helpPopup.addGoodSizedLabel("The symbol of your nation will appear next to the game when it's your turn".tr()).row()
+
+            helpPopup.addCloseButton()
+            helpPopup.open()
+        }
+        tab.add(helpButton)
+        tab.x = (stage.width - helpButton.width)
+        tab.y = (stage.height - helpButton.height)
+        stage.addActor(tab)
+
+        //TopTable Setup
+        //Have to put it into a seperate Table to be able to add another copyGameID button
+        val mainTable = Table()
+        mainTable.add(ScrollPane(leftSideTable).apply { setScrollingDisabled(true, false) }).height(stage.height*2/3)
+        mainTable.add(rightSideTable)
+        topTable.add(mainTable).row()
+        scrollPane.setScrollingDisabled(false, true)
+
+        //leftTable Setup
+        reloadGameListUI()
+
+        //Button to add currently running game to list if not yet done
+        val currentlyRunningGame = UncivGame.Current.gameInfo
+        if (currentlyRunningGame.gameParameters.isOnlineMultiplayer && !gameIsAlreadySavedAsMultiplayer(currentlyRunningGame.gameId)) {
+            topTable.add(
+                    TextButton("Add Currently Running Game".tr(), skin).onClick {
+                        if (!gameIsAlreadySavedAsMultiplayer(currentlyRunningGame.gameId)){
```

Comment:
Reverse the if to reduce indentation

Reply:
made it its own function now for better readability + reversed ifs to reduce indentation